api/v1alpha1/contentsourcebinding_types.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ContentSourceReference contains info to locate a Kind ContentSource object
+type ContentSourceReference struct {
+	// API version of the referent.
+	APIVersion string `json:"apiVersion,omitempty"`
+	// Kind is the type of resource being referenced.
+	Kind string `json:"kind,omitempty"`
+	// Name is the name of resource being referenced.
+	Name string `json:"name"`
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Namespaced,shortName=contentsourcebinding
+// +kubebuilder:printcolumn:name="ContentSource",type="string",JSONPath=".contentSourceRef.name"
+
+// ContentSourceBinding is a binding object responsible for
+// defining a ContentSource and a Namespace associated with it
+type ContentSourceBinding struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
```

Comment:
Since ContentSourceBinding is a root object, it should have a Spec (and optionally a Status), see other CRDs.

Reply:
Resolving as per offline discussion, I don't feel strongly either this or using a spec/status